feature-profile/src/main/java/com/codandotv/streamplayerapp/feature_profile/profile/data/ProfilePickerStreamRepository.kt

```diff
@@ -8,11 +8,13 @@ import com.codandotv.streamplayerapp.feature_profile.profile.domain.toProfiles
 import kotlinx.coroutines.flow.Flow
 import kotlinx.coroutines.flow.flowOf
 import kotlinx.coroutines.flow.map
+import org.koin.core.annotation.Factory
 
 interface ProfilePickerStreamRepository {
     suspend fun getProfiles(): Flow<List<ProfileStream>>
 }
 
+@Factory
```

Comment:
**#question**
Can we use the `@Factory` just inside of the classes annotated with `@Module`?

I think it would be better we define the annotations about how objects will be created in the module at the same file where the module is defined. What do you think?

Ty!

Reply:
This scenario just will be possible if you manually want to declare a dependency as a function - but this is the opposite of use Koin Annotations. This feature just exists to handle the corner cases.

To continue using factory, single etc inside the module, it's better don't use Koin Annotations.

Reply:
Great. Thanks for you explanation!